Formula/teku.rb

```diff
@@ -0,0 +1,59 @@
+class Teku < Formula
+  desc "Java Implementation of the Ethereum 2.0 Beacon Chain"
+  homepage "https://docs.teku.consensys.net/"
+  url "https://github.com/ConsenSys/teku/archive/refs/tags/21.12.2.tar.gz"
+  sha256 "fdd2e23ee8228b0bcb883f3bdfb4c24b82b21346a0444309e933ff019d5c3f39"
+  license "Apache-2.0"
+
+  depends_on "gradle" => :build
+  depends_on "openjdk"
+
+  def install
+    system "gradle", "installDist"
+
+    libexec.install Dir["build/install/teku/*"]
+
+    (bin/"teku").write_env_script libexec/"bin/teku", Language::Java.overridable_java_home_env
+
+    (bin/"teku_brew_services").write <<~EOS
+      #!/bin/bash
+      exec #{bin}/teku $(<#{etc}/teku.args)
+    EOS
+
+    (buildpath/"teku.args").write <<~EOS
+      --data-path=#{var}/teku
+      --log-destination=FILE
+      --log-file=#{var}/log/teku.log
+      --rest-api-enabled
+    EOS
+
+    etc.install "teku.args"
```

Comment:
We do not want to run our own configuration here. We only ship what upstream ships

Reply:
I can remove `--rest-api-enabled`, but the rest is specific to how the application is run with `brew services`. Is it okay to leave `data-path` and `log-*` in?

Reply:
No, all of this needs to be removed. If it can't run in the default configuration it can't have `brew services support`.

Reply:
I see. I was using [prometheus](https://github.com/Homebrew/homebrew-core/blob/bae1ec534356c4afa485e797ab1883c48bdf8bd2/Formula/prometheus.rb#L42) as an example but I wasn't aware this was an exception.

Reply:
I've removed all defaults and replaced it with an empty YAML config file.

Reply:
Yeah, I'm not sure how Prometheus was ever accepted like that

Reply:
Hm, I've removed the `brew services` section for now. The empty-YAML hack doesn't work either, so this software is not adequate to be run in compliance with the services rules.